scripts/zones/The_Garden_of_RuHmet/mobs/Ixaern_DRG.lua

```diff
@@ -7,37 +7,124 @@ local ID = zones[xi.zone.THE_GARDEN_OF_RUHMET]
 ---@type TMobEntity
 local entity = {}
 
+local breathList =
+{
+    [xi.mobSkill.WING_THRUST]       = xi.mobSkill.PET_FLAME_BREATH,
+    [xi.mobSkill.AURORAL_WIND]      = xi.mobSkill.PET_GUST_BREATH,
+    [xi.mobSkill.IMPACT_STREAM]     = xi.mobSkill.PET_LIGHTNING_BREATH,
+    [xi.mobSkill.DEPURATION]        = xi.mobSkill.PET_HYDRO_BREATH,
+    [xi.mobSkill.CRYSTALINE_COCOON] = xi.mobSkill.PET_FROST_BREATH,
+    [xi.mobSkill.MEDUSA_JAVELIN]    = xi.mobSkill.PET_SAND_BREATH,
+}
+
+local function toggleBracelets(mob)
+    local braceletActive = mob:getLocalVar('braceletActive')
+
+    if braceletActive == 0 then
+        mob:setLocalVar('braceletTimer', GetSystemTime() + 30)
+        mob:setLocalVar('braceletActive', 1)
+        mob:setAnimationSub(2)
+        mob:addMod(xi.mod.ATTP, 30)
+        mob:addMod(xi.mod.ACC, 40)
+        mob:addMod(xi.mod.DELAYP, -33)
+    elseif braceletActive == 1 then
+        mob:setLocalVar('braceletTimer', GetSystemTime() + 80)
+        mob:setLocalVar('braceletActive', 0)
+        mob:setAnimationSub(1)
+        mob:delMod(xi.mod.ACC, 40)
+        mob:delMod(xi.mod.ATTP, 30)
+        mob:delMod(xi.mod.DELAYP, -33)
+    end
+end
+
 entity.onMobInitialize = function(mob)
     mob:setMobMod(xi.mobMod.IDLE_DESPAWN, 300)
 end
 
+entity.onMobSpawn = function(mob)
+    mob:setLocalVar('braceletTimer', GetSystemTime() + 80)
+    mob:addImmunity(xi.immunity.BIND)
+    mob:addImmunity(xi.immunity.LIGHT_SLEEP)
+    mob:addImmunity(xi.immunity.DARK_SLEEP)
+    mob:addImmunity(xi.immunity.GRAVITY)
+    mob:addImmunity(xi.immunity.STUN)
+    mob:addImmunity(xi.immunity.TERROR)
```

Comment:
Immunities do not ever change, so please add them to onMobInitialize. No need to set them every spawn.